tests/ijby/test-sort.py

```diff
@@ -1016,6 +1016,157 @@ def test_sort_double_negation():
     assert_equals(DT[:, :, sort(f.A)], RES4)
 
 
+#-------------------------------------------------------------------------------
+# Sort with positional value for NAs
+#-------------------------------------------------------------------------------
+
+def key_func(x, rev, na_pos):
+    if na_pos == "first":
+        return not rev if x is not None else rev
+    else:
+        return not rev if x is None else rev
+
+def sort_func(src, rev, na_pos):
+    res = sorted(src, key=lambda x: (key_func(x, rev, na_pos), x), reverse=rev)
```

Comment:
If you handle `na_pos="remove"` here, then the tests can be simplified (making them parametrized on the `na_position` argument):
```
if na_pos == "remove":
  src = [s for s in src if s is not None]
```